python/python/tests/test_async_client.py

```diff
@@ -3347,7 +3420,7 @@ async def test_multi_key_command_returns_cross_slot_error(
             redis_client.sdiffstore("abc", ["def", "ghi"]),
         ]
 
-        if not check_if_server_version_lt(redis_client, "7.0.0"):
+        if not await check_if_server_version_lt(redis_client, "7.0.0"):
```

Comment:
How it worked before?

Reply:
From the Python docs:
> In the context of Boolean operations, and also when expressions are used by control flow statements, the following values are interpreted as false: False, None, numeric zero of all types, and empty strings and containers (including strings, tuples, lists, dictionaries, sets and frozensets). All other values are interpreted as true.

Before this change, the call to the function (without the await) was returning a reference to a Couroutine object, so this branch was never actually being entered.

Reply:
So test never tested 🤦 🤦 🤦